services/search-builder.js

```diff
@@ -138,7 +138,7 @@ function SearchBuilder(model, opts, params, fieldNamesRequested) {
           columnName = field.columnName || field.field;
 
           condition = opts.sequelize.where(
-            lowerIfNecessary(opts.sequelize.col(schema.name + '.' + columnName)),
+            lowerIfNecessary(opts.sequelize.col((params.associationName || schema.name) + '.' + columnName)),
```

Comment:
Could you explain the reason of those changes too 🙏

Reply:
The where parameters is created by this file, the problem is that it doesn't take aliases into account. So I'm just making sure if there is an lias it has the priority on the schema name